README.md

```diff
@@ -1 +1,29 @@
 # n-newsletter-signup
+
+## Styling Properties
+
+*  `theme`
+    *  Currently only `dark`. Adds a class modifier that can override the default style. `dark` allows for better usage on a dark background.
+*  `hideEmailLabel`
```

Comment:
I discussed this particular aspect with Dora before. I'm nervous of allowing too many fine grained options. I suggested having a `isStandalone` option to enable the email label (which has the benefit of being extensible to turn on other 'not part of a page full of signup components' features)

That would technically be a breaking change, but if we carefully coordinate we could add that property to our data model in the article app, so that when such a change gets merged the email text will still show up for us.

Reply:
Since this will be a major bump, as long as we document properly it should hopefully minimise any issues. Happy to change this to `isStandalone` which when enabled will show the email icon and append `Email` to the name?

Reply:
okey doke.